CONTRIBUTING.rst

```diff
@@ -41,6 +41,24 @@ How to Contribute to the Anaconda Installer (the short version)
 
 If you want to contribute a change to both the upstream and RHEL Anaconda then follow both a) and b) separately.
 
+Finding Bugs to Fix
+-------------------
+
+The development team can mark bugs with specific keywords to show that they belong to a specific
+category. You can quickly list these by searching the Red Hat bugzilla for bugs in the
+``anaconda`` component with specific keywords in Whiteboard:
+
+- For good first issues and simple fixes, the keyword is `EasyFix <https://bugzilla.redhat.com/buglist.cgi?bug_status=NEW&classification=Fedora&component=anaconda&f1=status_whiteboard&list_id=11496717&o1=substring&product=Fedora&query_format=advanced&v1=EasyFix>`_.
+
+- For Btrfs-related issues, use keyword `Btrfs <https://bugzilla.redhat.com/buglist.cgi?bug_status=NEW&classification=Fedora&component=anaconda&f1=status_whiteboard&list_id=11496717&o1=substring&product=Fedora&query_format=advanced&v1=Btrfs>`_.
+
+- For issues with `pure community features <pure-community-features>`_ (not supported by the development team), search for `CommunityFeature <https://bugzilla.redhat.com/buglist.cgi?bug_status=NEW&classification=Fedora&component=anaconda&f1=status_whiteboard&list_id=11496717&o1=substring&product=Fedora&query_format=advanced&v1=CommunityFeature>`_.
```

Comment:
I would prefer something like `For issues that are good candidates for pure community features, search for CommunityFeature`. However, it is up to you.

Reply:
I like this wording better over what's in the file too.

Reply:
Yes, that's better. Changed.